src/cli/simple/command/mod.rs

```diff
@@ -846,8 +871,13 @@ impl ContractCall<'_> {
             FinalExecutionStatus::Failure(e) => {
                 anyhow::bail!("{}: {e}", self.error_message)
             }
-            FinalExecutionStatus::SuccessValue(_) => match context.output_format {
-                OutputFormat::Plain => println!("{}", self.success_message),
+            FinalExecutionStatus::SuccessValue(output) => match context.output_format {
+                // TODO: The output could be serialized with JSON or Borsh.
+                // TODO: In the case of Borsh we should provide a type for deserializing the output in the corresponding object.
```

Comment:
Will this TODO be handled in a future PR?

Reply:
I hope so